src/app/common/hooks/use-submit-stx-transaction.ts

```diff
@@ -19,13 +19,17 @@ const timeForApiToUpdate = 250;
 
 interface UseSubmitTransactionArgs {
   loadingKey: string;
+  isIncreaseFeeTransaction?: boolean;
 }
 interface UseSubmitTransactionCallbackArgs {
   replaceByFee?: boolean;
   onSuccess(txId: string): void;
   onError(error: Error | string): void;
 }
-export function useSubmitTransactionCallback({ loadingKey }: UseSubmitTransactionArgs) {
+export function useSubmitTransactionCallback({
+  loadingKey,
+  isIncreaseFeeTransaction,
```

Comment:
Not big on this API decision here. What if we want to show another toast for a contract call transaction? Do we add `isContractCallTransaction:boolean`?

By adding a callback, or a custom toast message, we don't couple `useSubmitTransactionCallback` to a specific use case of it.

IMO the toast should be outside this function and called within the `onSuccess()` handler. Or at least, `isSuccessfulToastMessage` param.